neutrino.go

```diff
@@ -1250,6 +1261,31 @@ func (s *ChainService) handleAddPeerMsg(state *peerState, sp *ServerPeer) bool {
 		s.addrManager.Good(sp.NA())
 	}
 
+	// We'll go through each peer subscriber and notify it about the added
+	// peer.
+	n := 0
+	for i, sub := range s.peerSubscribers {
+		select {
+		case sub.peers <- sp:
```

Comment:
Should we be concerned about this blocking the main handler of the `ChainService` if a subscriber isn't ready to read from the channel?

Reply:
Idea was that it wasn't a problem blocking the peer handler, but one client stopping reading from its subscription will block the whole thing, which is obviously not good. Fixed by sending in a goroutine: https://github.com/lightninglabs/neutrino/pull/179/commits/1cdadade3d121aadf0ee999dbd515854f2eafd46